### ISSUE TITLE
how to generate outer json properties?

### ISSUE THREAD

**Issue by jbasrai** [ambitioninc/container-transform#14]
When I run container-transform on my docker-compose.yml, I only get the array for *containerDefinitions* generated.

**docker-compose.yml**
```
runner:
    image: ceroic/scrapyd
    ports:
        - "6800:6800"
    environment:
        - CLOSESPIDER_ITEMCOUNT=20
        - DOWNLOAD_DELAY=10
        - ELASTICSEARCH_SERVER=localhost
        - ELASTICSEARCH_PORT=9200
```

translated to:

```
[
    {
        "environment": [
            {
                "name": "CLOSESPIDER_ITEMCOUNT", 
                "value": "1"
            }, 
            {
                "name": "ELASTICSEARCH_PORT", 
                "value": "9200"
            }, 
            {
                "name": "DOWNLOAD_DELAY", 
                "value": "2"
            }, 
            {
                "name": "ELASTICSEARCH_SERVER", 
                "value": "localhost"
            }
        ], 
        "portMappings": [
            {
                "containerPort": 6800, 
                "hostPort": 6800
            }
        ], 
        "image": "ceroic/scrapyd", 
        "name": "runner", 
        "essential": true
    }
]
```
 
What do I have to change to get the whole ECS task definition to be generated, including *family* and *containerDefinitions* key?

**Comment by micahhausler** [comment]
Hey, I've got an alpha out right now that does that and haven't released that yet. You can try it out with:
```
pip install container-transform==0.5.0a2
```
and that should work!

**Comment by jbasrai** [comment]
So awesome.  Thanks for this great tool!

**Comment by micahhausler** [comment]
I released 0.5.0 today, let me know if you have any issues!
